sis-local/README.md

```diff
@@ -0,0 +1,63 @@
+# sis-local
+
+Emulate SiS functionality in a local environment. This may not work perfectly, but it should catch many common errors
+and provide a much smoother experience doing local development and porting to SiS!
+
+- Throw warnings for commands that don't work in SiS
+- (optional) Handles local Snowpark session setup
+
+**If you find some missing compatibility issue, please file an issue and tag `@sfc-gh-jcarroll`**
+
+## Usage
+
+Just like Snowpark, **it requires python 3.8**
+
+```shell
+pip install "streamlit_in_snowflake @ git+https://github.com/sfc-gh-jcarroll/sis-tricks.git@sis-local#subdirectory=sis-local"
+```
+
+### Magic code block for the top of your app
+
+```python
+from snowflake.snowpark.context import get_active_session
+from snowflake.snowpark.exceptions import SnowparkSessionException
+
+
+def in_snowpark() -> bool:
+    try:
+        sess = get_active_session()
+        return True
+    except SnowparkSessionException:
+        return False
+
+
+if in_snowpark():
+    import streamlit as st
+    session = get_active_session()
+else:
+    import streamlit_in_snowflake as st
+    # This part is optional, and requires secrets.toml setup
+    session = st.LocalSnowparkConnection().connect()
```

Comment:
Why `in_snowpark`?  Did you mean `in_sis()`? If the intended usecase for this block of code is to check whether one develops locally or in SiS, I'd just use something like this pattern instead maybe to make it clearer?

```
if developing_in_sis():
    ...
elif developing_locally():
    ...
else:
    raise Exception(...)
```

Reply:
I agree on renaming in_snowpark with in_sis, though I don't think you need a second test for developing_locally